content/java/concepts/array-list/terms/sort/sort.md

```diff
@@ -0,0 +1,104 @@
+---
+Title: '.sort()'
+Description: 'Used to sort arrays of primitive types and objects.'
+Subjects:
+  - 'Code Foundations'
+  - 'Computer Science'
+Tags:
+  - 'Arrays'
+  - 'Data Types'
+CatalogContent:
+  - 'learn-java'
+  - 'paths/computer-science'
+---
+
+The **`.sort()`** method is used to sort arrays of primitive types and objects. The sorting is done in ascending order by default, but it can be customized for objects by implementing the `Comparable` interface or using a `Comparator`.
+
+## Syntax
+
+Here's a basic syntax of how to use the `.sort()` method:
+
+```pseudo
+Arrays.sort(array);
+```
+
+## Examples
+
+### Example 1
+
+In this example, the `Arrays.sort(numbers)` method is used to sort the array of integers in ascending order. The result is then printed using `Arrays.toString()`.
+
+```java
+import java.util.Arrays;
+
+public class Main {
+  public static void main(String[] args) {
+    int[] numbers = {5, 2, 8, 1, 3};
+
+    // Sorting the array in ascending order
+    Arrays.sort(numbers);
+    
+    System.out.println("Sorted Array: " + Arrays.toString(numbers));
+  }
+}
+```
+
+The above example will give the following output:
+
+```shell
+Sorted Array: [1, 2, 3, 5, 8]
+```
+
+> **Note:** For an array of objects, you can use a custom comparator or ensure that the objects in the array implement the `Comparable` interface. Here's an example with strings.
```

Comment:
Avoid using first- and second-person pronouns (e.g. "I", "we", "you") if possible.